### PR TITLE
feat(locale zh): adds missing translations PE-1463

### PR DESCRIPTION
**Pull request by matibat**
Co-authored-by: Suning Yao <sy2940@nyu.edu>

### CODE REVIEW

**Review by thiagocarvalhodev — APPROVED** [review]

File: lib/l10n/app_zh.arb [hunk 1]
```diff
@@ -375,43 +375,43 @@
...
   "@rootFolderTxID": {
     "description": "The transaction ID of the Root Folder entity"
   },
-  "copyRootFolderTxID": "复制Root folder Tx ID",
+  "copyRootFolderTxID": "复制根文件夹交易ID",
   "@copyRootFolderTxID": {
     "description": "Copy transaction ID of Root Folder tooltip"
   },
-  "bundleTxID": "Bundle Tx ID",
+  "bundleTxID": "打包交易ID",
```
Comment (matibat) [inline lib/l10n/app_zh.arb:394]: Glad we have a translation for "Bundle"! We were struggling with that word in Spanish 😅

File: lib/l10n/app_zh.arb [hunk 2]
```diff
@@ -1039,7 +1039,7 @@
       }
     }
   },
-  "syncProgressPercentage": "{percentage}% complete",
+  "syncProgressPercentage": "{percentage}% 完成度",
```
Comment (matibat) [inline lib/l10n/app_zh.arb:1042]: Hey @fewwwww, just to confirm: it the percentage sign ("%") correctly used here?

File: lib/l10n/app_zh.arb (hunk 1 above)
Comment (fewwwww) [reply lib/l10n/app_zh.arb:394]: I really struggle on that too! Hard to find the perfect word for that.

File: lib/l10n/app_zh.arb (hunk 2 above)
Comment (matibat) [reply lib/l10n/app_zh.arb:1042]: Got ya! Thanks

**Comment by matibat** [comment]
Thank you again, @fewwwww <3
The team is gonna test the latest changes applied at https://github.com/ardriveapp/ardrive-web/pull/520 and release them as soon as we get the green flag! 😎

**Pull request merged**